cloud/scope/clients.go

```diff
@@ -158,19 +175,22 @@ func createClients(region string, oCIAuthConfigProvider common.ConfigurationProv
 	}, err
 }
 
-func createVncClient(region string, ociAuthConfigProvider common.ConfigurationProvider, logger *logr.Logger) (*core.VirtualNetworkClient, error) {
+func (c *ClientProvider) createVncClient(region string, ociAuthConfigProvider common.ConfigurationProvider, logger *logr.Logger) (*core.VirtualNetworkClient, error) {
 	vcnClient, err := core.NewVirtualNetworkClientWithConfigurationProvider(ociAuthConfigProvider)
 	if err != nil {
 		logger.Error(err, "unable to create OCI VCN Client")
 		return nil, err
 	}
 	vcnClient.SetRegion(region)
+	if c.ociClientOverrides != nil && c.ociClientOverrides.VCNClientUrl != nil {
+		vcnClient.Host = *c.ociClientOverrides.VCNClientUrl
+	}
 	vcnClient.Interceptor = setVersionHeader()
 
 	return &vcnClient, nil
 }
 
-func createNLbClient(region string, ociAuthConfigProvider common.ConfigurationProvider, logger *logr.Logger) (*networkloadbalancer.NetworkLoadBalancerClient, error) {
+func (c *ClientProvider) createNLbClient(region string, ociAuthConfigProvider common.ConfigurationProvider, logger *logr.Logger) (*networkloadbalancer.NetworkLoadBalancerClient, error) {
 	nlbClient, err := networkloadbalancer.NewNetworkLoadBalancerClientWithConfigurationProvider(ociAuthConfigProvider)
```

Comment:
we have to override here as well, we need tro add an override for nlb as well I think